checks/kubernetes/README.md

```diff
@@ -29,6 +29,19 @@ wget https://github.com/aquasecurity/trivy-checks/raw/main/test/testdata/kuberne
 conftest test denied.yaml --policy myPolicy/ --namespace builtin.kubernetes.KSV008
 ```
 
+# Kubernetes checks classification
+There are several Kubernetes checks that target various subsystems. They are loosely classified as follows.
+
+| Target         | Description                                                                                                                        |
+|----------------|------------------------------------------------------------------------------------------------------------------------------------|
+| Network        | Checks primarily targeting the networking stack                                                                                    |
+| Dynamic        | Checks that evaluate deprecated and removed APIs                                                                                   |
+| CIS Benchmarks | Checks that are recommended by the CIS Benchmarks. The checks inside are targeted per each subsystem (e.g. apiserver, cni, etc.)   |
```

Comment:
If this dir is serving only CIS, and CIS is using only this dir then fine, but I will challange us abit here: aren't these checks valid k8s checks also outside CIS context? aren't CIS reports include k8s checks besides these checks (i presume something like non privileged pod check).

Reply:
You're right in the sense that these checks are ordinary checks even outside of CIS context. 

I'm open to re-organizing all k8s checks in general (maybe based on another type) but we just need to confirm first with the stakeholders that this won't cause any breakage as in the past some users did rely on the structure. Can we do this in another PR?

Reply:
yes I think we need to reorganize the checks which is part of the motivation for this PR. sure if you think it needs another pr that's fine

Reply:
Sure, as we discussed offline, I've updated: https://github.com/aquasecurity/trivy-checks/pull/308/commits/271a630bfe0f1d7192276c71b2575e2d49b635e0